css/main.css

```diff
@@ -0,0 +1,402 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#logo {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+.intro,
+.about-me {
+  background-color: #1c1a19;
+  color: white;
+  position: relative;
+}
+
+.intro {
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+.intro-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.intro-greet,
+.about-me h2 {
+  color: #ff6b00;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.intro-me {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  align-items: center;
+  color: #fff5e1;
+}
+
+.intro-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social {
+  display: flex;
+}
```

Comment:
Please I suggest you use [DRY](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md) code (as mentioned by the previous reviewer). You can create reusable classes like `flex-center`, `row` and `column` to avoid code repetition. You can also use inheritance which allows you to apply the same styles to multiple elements by defining a parent element and then applying styles to its children. 

Please this is an example: 

```
.container {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
}

.social {
  @extend .container
}
```
This approach will make your code clean and maintainable. For more information click [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)